src/Footer/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import universalConfig from '../universal-config';
+import Logo from 'img/shared/logo.svg';
+
+import classnames from 'classnames';
```

Comment:
nit: to the top since it's a lib right?

Reply:
@jjwill Yes but it's related to styles. I think I got this habit from how docusaurus organized things as well. Personally I like it but I can change it if you don't